xbmc/cores/VideoPlayer/DVDMessage.h

```diff
@@ -282,6 +283,18 @@ class CDVDMsgOpenFile : public CDVDMsg
   FileParams m_params;
 };
 
+class CDVDMsgPlayerSetUpdateStreamDetails : public CDVDMsg
```

Comment:
live is much easier if you create 2 signals instead of having a single one with an additional paramter. this class is not needed if the only parameter is a string value